pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -282,4 +284,17 @@ private CompletableFuture<Void> validateClustersAsync(TenantInfo info) {
             }
         });
     }
+
+    private CompletableFuture<Void> validateAdminRoleAsync(TenantInfoImpl info) {
+        // empty adminRole shouldn't be allowed
+        if (!info.getAdminRoles().isEmpty()) {
+            for (String adminRole : info.getAdminRoles()) {
+                if (StringUtils.isBlank(adminRole) || StringUtils.containsWhitespace(adminRole)) {
+                    log.warn("[{}] Failed to validate due to adminRole {} is blank or containsWhitespace", clientAppId(), adminRole);
+                    return FutureUtil.failedFuture(new RestException(Status.PRECONDITION_FAILED, "AdminRole can not be blank or containsWhitespace"));
+                }
+            }
+        }
+        return CompletableFuture.completedFuture(null);
```

Comment:
If adminRoles() is empty, it should also need to return failure

Reply:
i think it can be empty